src/main/service/manualUploadService.ts

```diff
@@ -115,4 +118,83 @@ export class ManualUploadService {
     const court = await courtService.getCourtByName(courtName);
     return {courtName: courtName, courtId: court?.courtId};
   }
+
+  public async uploadPublication(data: any, ISODateFormat: boolean): Promise<boolean> {
+    if (this.getFileExtension(data.fileName) === 'json') {
+      return await dataManagementRequests.uploadJSONPublication(
+        data,
+        this.generatePublicationUploadHeaders(this.formatPublicationDates(data, ISODateFormat)),
+      );
+    } else {
+      return await dataManagementRequests.uploadPublication(
+        data,
+        this.generatePublicationUploadHeaders(this.formatPublicationDates(data, ISODateFormat)),
+      );
+    }
+  }
+
+  public removeFile(file): void {
+    const filePath = `./manualUpload/tmp/${file}`;
+    try {
+      fs.unlinkSync(filePath);
+    } catch (err) {
+      console.error(`Error while deleting ${file}.`);
+    }
+  }
+
+  public readFile(fileName): object {
+    try {
+      if (this.getFileExtension(fileName) === 'json') {
+        const rawData = fs.readFileSync(`./manualUpload/tmp/${fileName}`, 'utf-8');
+        return JSON.parse(rawData);
+      } else {
+        return fs.readFileSync(`./manualUpload/tmp/${fileName}`);
+      }
+    } catch (err) {
+      console.error(`Error while reading the file ${err}.`);
+      return null;
+    }
+  }
+
+  public joinDateValues(formData: any, dateName: string): string {
```

Comment:
this method already exists called `buildDate` line 84~

Reply:
true, didn't see that, removed